application/src/main/java/uk/nhs/hee/tis/revalidation/service/DoctorsForDBService.java

```diff
@@ -218,14 +216,4 @@ private long getCountUnderNotice() {
   private String getConnectionStatus(final String designatedBody) {
     return (designatedBody == null || designatedBody.equals("")) ? "No" : "Yes";
   }
-
-  private String getHiddenDesignatedBodyCode(String dbCode) {
-    if(dbCode != null && !dbCode.startsWith(hiddenPrefix)){
-      return hiddenPrefix + dbCode;
```

Comment:
Might be the code smell but it looks like there are no uses of `hiddenPrefix` any more.